tools/server/webui/src/routes/+layout.svelte

```diff
@@ -173,4 +177,4 @@
 	</div>
 </Sidebar.Provider>
 
-<svelte:window onkeydown={handleKeydown} />
+<svelte:window on:keydown={handleKeydown} bind:innerHeight />
```

Comment:
```suggestion
<svelte:window onkeydown={handleKeydown} bind:innerHeight />
```

Old syntax creeped in ;)